setup.py

```diff
@@ -5,26 +5,25 @@
 
 
 setup(
-    name="change_this_to_name_of_package",
-    version="0.1",
-    description="CREATE A DESCRIPTION",
-    url="https://github.com/CuriBio/CHANGE_THIS_TO_NAME_OF_REPO",
+    name="misc_test_utils",
+    version="0.2",
+    description="Persisting objects.",
+    url="https://github.com/CuriBio/misc-test-utils",
     author="Curi Bio",
     author_email="contact@curibio.com",
     license="MIT",
     packages=find_packages("src"),
     package_dir={"": "src"},
-    install_requires=[],
+    install_requires=["pytest>=5.4.3", "secrets_manager>=0.4", "domain-model==0.3"],
```

Comment:
should `secrets_manager` have a hyphen instead of underscore?

Reply:
I think either technically works, but yes it should change to be more consistent with our package names.